test/lan_test.rb

```diff
@@ -7,6 +7,26 @@
 Yast.import "Lan"
 
 describe "LanClass#Packages" do
+  it "does not list wpa_supplicant package when WIRELESS_AUTH_MODE is not psk or eap" do
```

Comment:
time to clean up it a little and reuse some mocks? what about start adding some context and use let to define some of the attributes?